pkg/ddevapp/config.go

```diff
@@ -1085,8 +1085,8 @@ RUN (groupadd --gid $gid "$username" || groupadd "$username" || true) && (userad
 	if extraPackages != nil {
 		contents = contents + `
 ### DDEV-injected from webimage_extra_packages or dbimage_extra_packages
-RUN echo "Disk usage problems: " && df -h /tmp && ls -l /var/lib/apt/lists
-RUN apt-get -qq update && DEBIAN_FRONTEND=noninteractive apt-get -qq install -y -o Dpkg::Options::="--force-confold" --no-install-recommends --no-install-suggests ` + strings.Join(extraPackages, " ") + "\n"
+RUN echo "Disk usage problems: " && df -h / && ls -l /var/lib/apt/lists
```

Comment:
@rfay I'm not sure if we still need this check, if we skip failures for `apt-get update`, but anyway:
I changed `df -h /tmp` to `df -h /` - we discussed this on Discord earlier and `df -h /tmp` produced less output.

Reply:
Agreed we probably won't need it at all, can remove. Install of PHP related packages can still fail, but we have no control. And this check didn't actually do anything but somehow seemed to result in less failures